src/core/crypto/AESEncryptionService.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Password } from '../../model/wallet/Password';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const CryptoJS = require('crypto-js');
+
+export class AESEncryptionService {
```

Comment:
- If this service is only used by wallet I would call it something else e.g `SimpleWalletService`
- Otherwise, to make it generic, remove wallet reference (import), also considering to merge it into the Crypto.ts class to replace the existing `encrypt / decrypt` functions.

I would prefer the latter option.